examples/experimental/slpk-in-browser/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "slpk-in-browser1",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@deck.gl/core": "^8.9.28",
+    "@deck.gl/extensions": "^8.9.28",
+    "@deck.gl/geo-layers": "^8.9.28",
+    "@deck.gl/layers": "^8.9.28",
+    "@deck.gl/mesh-layers": "^8.9.28",
+    "@deck.gl/react": "^8.9.34",
+    "@loaders.gl/core": "^4.0.0",
+    "@loaders.gl/schema": "^4.0.0",
+    "@loaders.gl/i3s": "^4.0.0",
+    "@loaders.gl/loader-utils": "^4.0.0",
+    "@loaders.gl/mvt": "^4.0.0",
+    "@loaders.gl/terrain": "^4.0.0",
+    "@testing-library/jest-dom": "^5.17.0",
```

Comment:
That is a LOT of dependencies. Do we really need the test libraries for an example? Can they go into `devDependencies`?